[2024-05-14T03:30:34.871+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-14T03:30:34.896+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: MI_PRIMER_DAG.first_task manual__2024-05-14T03:30:31.379118+00:00 [queued]>
[2024-05-14T03:30:34.906+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: MI_PRIMER_DAG.first_task manual__2024-05-14T03:30:31.379118+00:00 [queued]>
[2024-05-14T03:30:34.907+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 5
[2024-05-14T03:30:34.927+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): first_task> on 2024-05-14 03:30:31.379118+00:00
[2024-05-14T03:30:34.934+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1694) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-14T03:30:34.936+0000] {standard_task_runner.py:63} INFO - Started process 1698 to run task
[2024-05-14T03:30:34.936+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'MI_PRIMER_DAG', 'first_task', 'manual__2024-05-14T03:30:31.379118+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/mi_primer_dag.py', '--cfg-path', '/tmp/tmpcxyr43on']
[2024-05-14T03:30:34.940+0000] {standard_task_runner.py:91} INFO - Job 29: Subtask first_task
[2024-05-14T03:30:34.999+0000] {task_command.py:426} INFO - Running <TaskInstance: MI_PRIMER_DAG.first_task manual__2024-05-14T03:30:31.379118+00:00 [running]> on host 33978f229c7f
[2024-05-14T03:30:35.098+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='MI_PRIMER_DAG' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-14T03:30:31.379118+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-14T03:30:31.379118+00:00'
[2024-05-14T03:30:35.101+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-14T03:30:35.101+0000] {logging_mixin.py:188} INFO - Hola Mundo
[2024-05-14T03:30:35.102+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-14T03:30:35.102+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-14T03:30:35.113+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=MI_PRIMER_DAG, task_id=first_task, execution_date=20240514T033031, start_date=20240514T033034, end_date=20240514T033035
[2024-05-14T03:30:35.151+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-14T03:30:35.194+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-14T03:30:35.254+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
